torchvision/transforms/autoaugment.py

```diff
@@ -0,0 +1,236 @@
+            If input is PIL Image, the options is only available for ``Pillow>=5.0.0``.
+
+    Example:
+        >>> t = transforms.AutoAugment()
+        >>> transformed = t(image)
+
+        >>> transform=transforms.Compose([
+        >>>     transforms.Resize(256),
+        >>>     transforms.AutoAugment(),
+        >>>     transforms.ToTensor()])
+    """
+
+    def __init__(self, policy: AutoAugmentPolicy = AutoAugmentPolicy.IMAGENET, fill: Optional[List[float]] = None):
+        super().__init__()
+        self.policy = policy
+        self.fill = fill
+        self.transforms = _get_transforms(policy)
+        if self.transforms is None:
+            raise ValueError("The provided policy {} is not recognized.".format(policy))
+
+        _BINS = 10
+        self._op_meta = {
+            # name: (magnitudes, signed)
+            "ShearX": (torch.linspace(0.0, 0.3, _BINS), True),
+            "ShearY": (torch.linspace(0.0, 0.3, _BINS), True),
+            "TranslateX": (torch.linspace(0.0, 150.0 / 331.0, _BINS), True),
+            "TranslateY": (torch.linspace(0.0, 150.0 / 331.0, _BINS), True),
+            "Rotate": (torch.linspace(0.0, 30.0, _BINS), True),
+            "Brightness": (torch.linspace(0.0, 0.9, _BINS), True),
+            "Color": (torch.linspace(0.0, 0.9, _BINS), True),
+            "Contrast": (torch.linspace(0.0, 0.9, _BINS), True),
+            "Sharpness": (torch.linspace(0.0, 0.9, _BINS), True),
+            "Posterize": (torch.tensor([8, 8, 7, 7, 6, 6, 5, 5, 4, 4]), False),
+            "Solarize": (torch.linspace(256.0, 0.0, _BINS), False),
+            "AutoContrast": (None, None),
+            "Equalize": (None, None),
+            "Invert": (None, None),
+        }
+
+    @staticmethod
+    def get_params(transform_num: int) -> Tuple[int, Tensor, Tensor]:
+        """Get parameters for autoaugment transformation
+
+        Returns:
+            params required by the autoaugment transformation
+        """
+        policy_id = torch.randint(transform_num, (1,)).item()
+        probs = torch.rand((2,))
+        signs = torch.randint(2, (2,))
+
+        return policy_id, probs, signs
+
+    def _get_op_meta(self, name: str) -> Tuple[Optional[Tensor], Optional[bool]]:
+        return self._op_meta[name]
+
+    def forward(self, img: Tensor):
+        """
+            img (PIL Image or Tensor): Image to be transformed.
+
+        Returns:
+            PIL Image or Tensor: AutoAugmented image.
+        """
+        fill = self.fill
+        if isinstance(img, Tensor):
+            if isinstance(fill, (int, float)):
+                fill = [float(fill)] * F._get_image_num_channels(img)
+            elif fill is not None:
+                fill = [float(f) for f in fill]
+
+        transform_id, probs, signs = self.get_params(len(self.transforms))
+
+        for i, (op_name, p, magnitude_id) in enumerate(self.transforms[transform_id]):
+            if probs[i] <= p:
```

Comment:
Nit: as you wish, for clarity and less indents can be
```python
if probs[i] > p:
    continue
```

Reply:
Good spot, I'll make the change.

Reply:
Sorry to sound like a broken record here but that's not supported by JIT:
```
E                   if probs[i] > p:
E                       continue
E                       ~~~~~~~~ <--- HERE
```